_episodes/02-meet-docker.md: add section on how to get help. (#100)
carpentries-incubator/docker-introduction: _episodes/02-meet-docker.md

objectives:
- "Explain how to check that Docker is installed and is ready to use."
- "Demonstrate some initial Docker command line interactions."

> - "Remember how to get help for Docker commands."

Reviewer: ```suggestion
- "Use the built-in help for Docker commands."
```
Think this might be better as the "apply" level of bloom's taxonomy.

then you need to check that you have started the Docker Desktop, Docker Engine, or however else you worked through the setup instructions.

## Getting help

> Whenever we are working with a new command line tool, we need to get help at some point. These tools often have some

Reviewer: ```suggestion
Often when working with a new command line tool, we need to get help. These tools often have some
```

> sort of subcommand or flag (usually `help`, `-h`, or `--help`) that displays a help prompt describing how to use the

Reviewer: ```suggestion
sort of subcommand or flag (usually `help`, `-h`, or `--help`) that displays a prompt describing how to use the
```

  stop        Stop one or more running containers
  top         Display the running processes of a container
  unpause     Unpause all processes within one or more containers
  update      Update configuration of one or more containers
  wait        Block until one or more containers stop, then print their exit codes

Run 'docker container COMMAND --help' for more information on a command.
~~~
{: .output}

There's also help for the `container ls` command:
~~~
docker container ls --help  # this one actually requires the '--help' flag
~~~
{: .language-bash}

~~~
Usage:  docker container ls [OPTIONS]

List containers

Aliases:
  ls, ps, list

Options:
  -a, --all             Show all containers (default shows just running)
  -f, --filter filter   Filter output based on conditions provided
      --format string   Pretty-print containers using a Go template
  -n, --last int        Show n last created containers (includes all states) (default -1)
  -l, --latest          Show the latest created container (includes all states)
      --no-trunc        Don't truncate output
  -q, --quiet           Only display container IDs
  -s, --size            Display total file sizes
~~~
{: .output}

> We can notice that there are many commands that stem from the `docker` command. Instead of trying to remember

Reviewer: ```suggestion
You may notice that there are many commands that stem from the `docker` command. Instead of trying to remember
```

all possible commands and options, it's better to learn how to effectively get help from the command line. Although

> we can always search the web, getting the built-in help from our tool is usually much faster and may provide the answer

Reviewer: ```suggestion
we can always search the web, getting the built-in help from our tool is often much faster and may provide the answer
```

> right away. This applies not only to Docker, but rather to most command line-based tools.

Reviewer: ```suggestion
right away. This applies not only to Docker, but also to most command line-based tools.
```
